Client/Frontend/UserContent/UserScripts/ReadyState.js

```diff
@@ -0,0 +1,73 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+(function() {
+  // Listen for document ready state
+  document.addEventListener('readystatechange', (event) => {
+    window.webkit.messageHandlers.$<handler>.postMessage({
+      "securitytoken": "$<security_token>",
```

Comment:
are we sure that there's no way webpages can see the token here using https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/toString or similar?

Reply:
Yes I am sure. I made sure to test for this, and all our existing scripts do the same.
Any script that exposes a function for a web-page has the `toString` function overridden to return `[native method]` as a string, and not the implementation.